testnet/stacks-node/src/burnchains/mocknet_controller.rs

```diff
@@ -99,13 +101,22 @@ impl BurnchainController for MocknetController {
     fn get_stacks_epochs(&self) -> Vec<StacksEpoch> {
         match &self.config.burnchain.epochs {
             Some(epochs) => epochs.clone(),
-            None => vec![StacksEpoch {
-                epoch_id: StacksEpochId::Epoch20,
-                start_height: 0,
-                end_height: STACKS_EPOCH_MAX,
-                block_limit: ExecutionCost::max_value(),
-                network_epoch: PEER_VERSION_EPOCH_2_0,
-            }],
+            None => vec![
+                StacksEpoch {
+                    epoch_id: StacksEpochId::Epoch20,
+                    start_height: 0,
+                    end_height: 1,
+                    block_limit: ExecutionCost::max_value(),
+                    network_epoch: PEER_VERSION_EPOCH_2_0,
+                },
```

Comment:
Hey, you're going to need to add an epoch here to transition the system to Epoch 2.05.  After that, you can transition to 2.1.

Reply:
Amended.